src/cr/cube/stripe.py

```diff
@@ -519,9 +576,9 @@ class _MeansMrStripeRow(_BaseStripeRow):
 
     def __init__(self, element, unweighted_counts, means):
         super(_MeansMrStripeRow, self).__init__(element)
-        # ---unweighted_counts is an int array like [336 5501]---
+        # ---unweighted_counts is a two-value int array like [336 5501]---
```

Comment:
Not sure why though... I can see how this comment would be important for the reader, but it lacks context. At least, I cannot figure it without research (into code) _why_ this should ever be exactly two values. Maybe add some context if it's not too verbose?